src/utils/clack-utils.ts

```diff
@@ -30,6 +31,42 @@ interface WizardProjectData {
   projects: SentryProjectData[];
 }
 
+export interface CliSetupConfig {
+  filename: string;
+  name: string;
+
+  likelyAlreadyHasAuthToken(contents: string): boolean;
+  tokenContent(authToken: string): string;
+
+  likelyAlreadyHasOrgAndProject(contents: string): boolean;
+  orgAndProjContent(org: string, project: string): string;
```

Comment:
l: Given that we want to move towards org-scoped auth tokens, we should probably split this up to org and project. Reason: With org-auth tokens our wizards don't need to set the org slug anymore. Sentry CLI should already support this. 

Feel free to skip this though because afaik no wizard actually leverages this at the moment. The Sentry backend returns org auth tokens for users who only belong to one org. For users with multiple orgs, we still return old auth tokens, due to the chicken-egg problem described in #357.

Reply:
yeah I think I'm gonna skip this for now, I think it better fits PR where we switch to use org auth tokens